src/extension/php_objects.h

```diff
@@ -26,26 +36,70 @@ typedef struct _dd_ini_setting {
     // or a pointer to int representing the tsrm offset
     void *global_variable;
     size_t field_offset;
+    const char *name_prefix;
```

Comment:
I wonder if perhaps you could've done all of this with just a flag (`bool tracer_setting`) and use the macros everywhere, I guess it doesn't matter now.

Reply:
It could have been done that way yes